src/Markdig/Markdown.cs

```diff
@@ -86,6 +86,28 @@ public static string ToHtml(string markdown, MarkdownPipeline pipeline = null, M
             return html;
         }
 
+        /// <summary>
+        /// Converts a Markdown document to HTML.
+        /// </summary>
+        /// <param name="document">A Markdown document.</param>
+        /// <param name="pipeline">The pipeline used for the conversion.</param>
+        /// <returns>The result of the conversion</returns>
+        /// <exception cref="ArgumentNullException">if markdown document variable is null</exception>
+        public static string ToHtml(MarkdownDocument document, MarkdownPipeline pipeline = null)
```

Comment:
```suggestion
        public static string ToHtml(this MarkdownDocument document, MarkdownPipeline pipeline = null)
```

Maybe make it extension methods as well by adding this?

Reply:
Yes, it looks better.